python-backend/app/api/mines/mine/resources/mine.py

```diff
@@ -232,20 +232,20 @@ def post(self, mine_no_or_guid=None):
             self.raise_error(400, 'Error: Unexpected mine number in Url.'), 400
 
         data = self.parser.parse_args()
-        lat = data['latitude']
-        lon = data['longitude']
-        note = data['note']
+        lat = data.get('latitude')
+        lon = data.get('longitude')
+        note = data.get('note')
         location = None
         mine_region = None
-        status = data['mine_status']
-        major_mine_ind = data['major_mine_ind']
-        mine_region = data['mine_region']
+        status = data.get('mine_status')
+        major_mine_ind = data.get('major_mine_ind')
+        mine_region = data.get('mine_region')
```

Comment:
The string indexing was a form of safety (crash if this doesn't exist). If this code continues with values being null (that were never null before). Are there potential issues? Are there now validators that none of these are None?

Reply:
My concern as well. I remember there being discussion around this, although I didn't actively research or weight in on it.

Reply:
So, perhaps revert the `.get()`s and wrap it in a `try` `except KeyError` so that we can handle it gracefully?

Reply:
The point of the change was to save boilerplate code though?

Reply:
Hmm 🤔 Your point makes sense. On one hand we risk crashing the app on other flushing the DB with null values.
I say we stick to crashing for now and we make a ticket to address the bigger issue here of handling exceptions properly.
The point of this PR was just a bit of housekeeping, I wasn't expecting to change any patterns or behavior